python/cuspatial/pyproject.toml

```
@@ -13,12 +13,98 @@
 # limitations under the License.
 
 [build-system]
-
+build-backend = "setuptools.build_meta"
 requires = [
-    "wheel",
-    "setuptools",
-    "cython>=0.29,<0.30",
-    "scikit-build>=0.13.1",
     "cmake>=3.23.1,!=3.25.0",
+    "cudf==23.6.*",
+    "cython>=0.29,<0.30",
     "ninja",
+    "rmm==23.6.*",
+    "scikit-build>=0.13.1,<0.17.2",
+    "setuptools",
+    "wheel",
+] # This list was generated by `rapids-dependency-file-generator`. To make changes, edit ../../dependencies.yaml and run `rapids-dependency-file-generator`.
+
+[project]
+name = "cuspatial"
+version = "23.6.0"
+description = "cuSpatial: GPU-Accelerated Spatial and Trajectory Data Management and Analytics Library"
+readme = { file = "README.md", content-type = "text/markdown" }
+authors = [
+    { name = "NVIDIA Corporation" },
+]
+license = { text = "Apache 2.0" }
+requires-python = ">=3.8"
```

We are dropping Python 3.8 in RAPIDS 23.06.
```suggestion
requires-python = ">=3.9"
```